packaging_automation/delete_packages_on_packagecloud.py

```diff
@@ -0,0 +1,57 @@
+import requests
+import json
+from datetime import datetime
+import argparse
+from enum import Enum
+
+PAGE_RECORD_COUNT = 100
+
+
+class PackageRepository(Enum):
+    community_nightlies = "community-nightlies"
+    enterprise_nightlies = "enterprise-nightlies"
+
+
+def delete_packages(repo: PackageRepository, package_cloud_api_token: str) -> None:
+    url_prefix = f"https://{package_cloud_api_token}:@packagecloud.io"
+
+    successful_count = 0
+    error_count = 0
+    end_of_limits_reached = False
+    while True:
+        list_url = (f"{url_prefix}/api/v1/repos/citusdata/{repo.value}"
+                    f"/packages.json?per_page={PAGE_RECORD_COUNT}&page=0")
+        result = requests.get(list_url)
+        package_info_list = json.loads(result.content)
+        if len(package_info_list) == 0 or end_of_limits_reached:
+            break
+        for package_info in package_info_list:
+            package_upload_date = datetime.strptime(package_info['created_at'], "%Y-%m-%dT%H:%M:%S.000Z")
+            diff = datetime.now() - package_upload_date
+            if diff.days > 10:
```

Comment:
maybe put 10 to a global variable as in PAGE_RECORD_COUNT:
`PACKAGE_DELETION_DAYS_THRESHOLD = 10`

Reply:
Fixed as you suggested